admin/client.py

```diff
@@ -1,176 +1,164 @@
 # Copyright Hybrid Logic Ltd.  See LICENSE file for details.
 """
-Run the acceptance tests.
+Run the client installation tests.
 """
 
+import os
+import shutil
 import sys
+import tempfile
 import yaml
 
-from zope.interface import Interface, implementer
-from characteristic import attributes
-from eliot import add_destination
-from twisted.internet.error import ProcessTerminated
+import docker
+from effect import TypeDispatcher, sync_performer, perform
 from twisted.python.usage import Options, UsageError
 from twisted.python.filepath import FilePath
-from twisted.internet.defer import inlineCallbacks, returnValue
 
-from flocker.common.version import make_rpm_version
-from flocker.provision import PackageSource, CLOUD_PROVIDERS
 import flocker
-from flocker.provision._ssh import (
-    run_remotely)
+from flocker.common.version import make_rpm_version
+from flocker.provision import PackageSource
+from flocker.provision._effect import Sequence, perform_sequence
```

Comment:
Maybe it'd be good to have a look at how much time https://clusterhq.atlassian.net/browse/FLOC-2810 would take, and consider whether that time is worth it to avoid building so much on `Sequence` which we want to get rid of - merging this PR will make that take longer.